_appmap/test/test_flask.py

```diff
@@ -30,6 +31,8 @@ def flask_app(data_dir, monkeypatch):
 
     Env.current.set("APPMAP_CONFIG", data_dir / "flask" / "appmap.yml")
 
+    monkeypatch.setenv("FLASK_DEBUG", "True")
+
```

Comment:
It seems instead of setting `@pytest.mark.appmap_enabled` in each test, `FLASK_DEBUG` is set in flask_app.

Reply:
Yes, this depends on record-by-default working for a Flask app running with `FLASK_DEBUG` set to `True`. Is that what you meant? Do you see a problem here?

Reply:
This is what I meant.  I wanted to make sure I understood it.

As long as it's possible to run tests with `FLASK_DEBUG=False` I don't see a problem.

Reply:
> As long as it's possible to run tests with FLASK_DEBUG=False I don't see a problem.
Well, you can't change this code to say `monkeypatch.setenv("FLASK_DEBUG", "False")`, that would cause tests to fail. If you wanted to set `FLASK_DEBUG=False` in your shell before you ran the tests, though, this would override it, so the tests would pass.